movie-app/src/constants/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const useQuery = (locationSearch: string) => {
```

Comment:
it doesn't seem like a custom hook: this method doesn't  use any other hooks in the shell. I'd recommend to add here `useLocation` and `useMemo` here